tests/helpers/test_resource_not_in_the_list.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this test can be removed

Reply:
You should leave this as it hints the Python interpreter to read the file in UTF-8 which is a good safety practice when dealing with a 2.7/3.x program.

Reply:
Sorry, by 'this' I meant the file